docs/dev/testing.rst

```diff
@@ -120,9 +120,10 @@ or:
 
 Full ETL Tests
 ^^^^^^^^^^^^^^
-As mentioned above the CI tests process a single year of data. If you would like to
-more exhaustively test the ETL process without affecting your existing FERC 1 and PUDL
-databases, you can use the ``full`` test environment:
+As mentioned above the CI tests process a single year of data. If you would
+like to more exhaustively test the ETL process without affecting your
```

Comment:
Would be nice to let users know how long this might take.

Reply:
Done.